### PR TITLE
Fix/tests

### PR DESCRIPTION
**Pull request by AnnRiera**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/blockcoders/near-rpc-providers/blob/main/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: src/near-rpc-provider.spec.ts [hunk 1]
```diff
@@ -28,6 +33,15 @@ describe('NearRpcProvider', () => {
...
     })
+
+    it('should throw an error if params are not provided', async () => {
+      try {
+        await provider.getBalance('')
+      } catch (error) {
+        expect(error).to.exist
+        expect(error).to.be.an.instanceof(Error)
+      }
+    })
```
Comment (jarcodallo) [inline src/near-rpc-provider.spec.ts:44]: You can do this in one line like this:
```ts
await expect(() => provider.getBalance('')).to.be.rejectedWith(Error)
```

File: src/near-rpc-provider.spec.ts [hunk 2]
```diff
@@ -51,6 +65,15 @@ describe('NearRpcProvider', () => {
...
     })
+
+    it('should throw an error if params are not provided', async () => {
+      try {
+        await provider.getBlock('')
+      } catch (error) {
+        expect(error).to.exist
+        expect(error).to.be.an.instanceof(Error)
+      }
+    })
```
Comment (jarcodallo) [inline src/near-rpc-provider.spec.ts:76]: You can do this in one line like this:
```ts
await expect(() => provider.getBlock('')).to.be.rejectedWith(Error)
```

File: src/near-rpc-provider.spec.ts [hunk 3]
```diff
@@ -59,6 +82,15 @@ describe('NearRpcProvider', () => {
...
     })
+
+    it('should throw an error if there is something wrong', async () => {
+      try {
+        await provider.getGasPrice()
+      } catch (error) {
+        expect(error).to.exist
+        expect(error).to.be.an.instanceof(Error)
+      }
+    })
```
Comment (jarcodallo) [inline src/near-rpc-provider.spec.ts:93]: You can do this in one line like this:
```ts
await expect(() => provider.getGasPrice()).to.be.rejectedWith(Error)
```

File: src/near-rpc-provider.spec.ts [hunk 4]
```diff
@@ -99,6 +131,85 @@ describe('NearRpcProvider', () => {
...
+
+    it('should throw an error if params are not provided', async () => {
+      try {
+        await provider.getCode('', '')
+      } catch (error) {
+        expect(error).to.exist
+        expect(error).to.be.an.instanceof(Error)
+      }
+    })
+  })
```
Comment (jarcodallo) [inline src/near-rpc-provider.spec.ts:143]: You can do this in one line like this:
```ts
await expect(() => provider.getCode('', '')).to.be.rejectedWith(Error)
```

**Review by jarcodallo — CHANGES_REQUESTED** [review]
Use `.to.be.rejectedWith(Error)` helper to expect an error

File: src/near-rpc-provider.spec.ts (hunk 4 above)
Comment (AnnRiera) [reply src/near-rpc-provider.spec.ts:143]: Got it!

File: src/near-rpc-provider.spec.ts (hunk 3 above)
Comment (AnnRiera) [reply src/near-rpc-provider.spec.ts:93]: Roger that!

File: src/near-rpc-provider.spec.ts (hunk 2 above)
Comment (AnnRiera) [reply src/near-rpc-provider.spec.ts:76]: Copy!

File: src/near-rpc-provider.spec.ts (hunk 1 above)
Comment (AnnRiera) [reply src/near-rpc-provider.spec.ts:44]: Ok!

**Review by jarcodallo — APPROVED** [review]
LGTM

**Pull request merged**